### DISCUSSION TITLE
Is dark theme working again?

### DISCUSSION THREAD

**Discussion by Sebastian1989101** [gitextensions/gitextensions#10855] [General] [answered]
After seeing that dark theme was no option with 4.x, I rolled my Git Ext back to 3.x. I was just wondering if this is resolved by now because I could not find anything about it with a quick search. In 2023 I think it should be default to have a dark theme simply to not burn your eyes when opening a application in the dark..

**Comment by maraf** [comment, marked as answer]
The reason dark theme is missing is because it is simply not possible on .NET 6/7 WinForms at the moment. You can find some closed issues with deep discussion on this repo.

**Reply by Sebastian1989101** [reply]
Then, sorry if this sounds harsh, .NET 6/7 WinForms is simply not production rdy and should not be used for anyhting. Just because Microsoft throws something out does not mean anything these days. Just look at MAUI and it's horrible state.

**Reply by RussKie** [reply]
Your opinion is baseless ans frankly unwelcome.

**Reply by Sebastian1989101** [reply]
> Your opinion is baseless ans frankly unwelcome.

Baseless? It is baseless that Microsoft is releasing product after product unfinished bugged as hell? Unwelcome, I get it, based on your Profile you are in some way related to Microsoft. But baseless for sure not. A framework is supposed to make things easier, more accessable. Everything Microsoft has released for .NET in the last 2-3 years is just a bugged mess that makes it really hard. MAUI is the biggest joke in history due to bugs and issues. And if things in WinForms got broken without a fix but made it to production says enough about .NET.

**Reply by vbjay** [reply]
.net 6 is done well. What you are misunderstanding is the tech used that allowed us to wire up the hooks was removed in .net vs framework.  Because .net had lost this tech we can't wire up the hooks.  It was a foundational change that is not an easy change this and  make it work again change. 


Where you goofed. You have not looked at the issues tied to theming and all the conversations and work that was done to try to make it work. Maybe you could read them and get up to date before shooting from the hip with attacking posts. 

So yes your baseless, unnecessary, and vitriolic responses are not needed. @RussKie  is right.

**Reply by vbjay** [reply]
Yes. We all are bummed about it. We would LOVE to have theming work again. @NikolayXHD  did some really amazing work and we value his work. Will theming work again....maybe. We had somebody chime in recently that had a potential fix that hasn't been researched and looked into yet but will take a lot of vetting.

**Reply by vbjay** [reply]
https://learn.microsoft.com/en-us/dotnet/core/porting/net-framework-tech-unavailable
